AshleyLab/motrpac-frontend
# MOTR-251 Upgrade auth0-js dependency to its latest version plus other npm pkg version updates

**Key Changes:**

1. Upgraded `auth0-js` to its latest version due to found vulnerability.
2. In the process of applying minor update to `jquery`, its latest version `3.5.0` caused **Bootstrap**'s expand/collapse component to fail on our releases page. Hence `v3.4.1` has been pinned to it for now.
3. Also applied minor updates to other dependencies, such as `react`, `react-dom`, and `react-particles-js`.

***

**Steps to test:**

1. If you are testing this in an existing local working copy of the repo, do try to run `rm -r node_modules/ && rm yarn.lock && yarn install` prior to starting the test.
2. After the step above, run `yarn inspect-branch` and expect to be able to bring up the portal at `localhost:3000` without errors.
3. In a separate Terminal window, `cd` to the same local working copy of the repo, run `yarn test` and invoke the unit testing. Expect the test to run through without errors.
4. If the above step completes successfully, press <kbd>control</kbd> + <kbd>C</kbd> to exit unit testing.
5. Run `yarn storybook` and expect to be able to bring up **Storybook** in a browser window without errors.
6. Lastly, log into the portal and navigate to the Releases page. Expect no errors shown in the browser console (ignore the "SameSite" cookie warning if you are using Chrome).
